week2/homework/src/updateToDo.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const { loadToDos, saveToDo, logScreen } = require('./Util');
+
+const updateToDo = async (order, body) => {
+  const todos = await loadToDos();
+  const todo = todos.find(todo => todo.index === order);
+  if (todo) {
+    todos[order - 1] = { index: order, body, complete: '➖' };
```

Comment:
you could instead make it as what you have done with [completeTodo]:
```javascript 
todos[order - 1].body = 'stuff'
```

other than legibility, what if this todo is done, you are resetting it to be not done.

[completeTodo]: https://github.com/HackYourFuture/Node.js/pull/447/files#diff-c03001336cfcb3bf1db5d7fb92e1193aR8

Reply:
I have noticed it now, that is why reviewing the code by someone else important. :)